units/host_status.go

```diff
@@ -107,6 +107,10 @@ clientsLoop:
 					j.AddError(j.terminateUnknownHosts(ctx, err.Error()))
 					continue clientsLoop
 				}
+				grip.Debug(message.WrapError(err, message.Fields{
+					"message": "getting instance statuses from AWS",
```

Comment:
nit: this message here isn't wrapping an error (the way that errors.Wrap does) so it does make sense to start the sentence with "error" or "problem" to explain that this is an issue, not just that you _are_ getting the instance status.